05-05 02:56:02 INFO [0m[hpotato.core.plugins.base] Opening url: get https://m.media-amazon.com/images/M/MV5BM2M0ODQ0NDYtMGUzNi00MDdjLWI0MzYtY2RjYTdlYjk5NDFiXkEyXkFqcGdeQXVyMzM2OTQwMDY@._V1_SX300.jpg, data: [][0m
05-05 02:56:06 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/search/movie?api_key=xxx&search_type=phrase&query=the+prodigy&year=2019, data: [][0m
05-05 02:56:06 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/search/The+Prodigy+2019/?limit=1, data: [][0m
05-05 02:56:06 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.omdbapi.com/?apikey=xxx&type=movie&y=2019&t=the+prodigy, data: [][0m
05-05 02:56:06 INFO [0m[ie.providers.info.omdbapi] Found: The Prodigy (2019)[0m
05-05 02:56:06 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/info/tt4504044/?ignore=1, data: [][0m
05-05 02:56:06 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.omdbapi.com/?apikey=xxx&type=movie&i=tt4504044, data: [][0m
05-05 02:56:06 INFO [0m[ie.providers.info.omdbapi] Found: The Prodigy (2019)[0m
05-05 02:56:06 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt4504044?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:06 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/ismovie/tt4504044/, data: [][0m
05-05 02:56:07 INFO [0m[ie.providers.info.omdbapi] Found: The Prodigy (2019)[0m
05-05 02:56:07 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt4504044?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:07 INFO [0m[ie.providers.info.omdbapi] Found: The Prodigy (2019)[0m
05-05 02:56:07 INFO [0m[hpotato.core.plugins.base] Opening url: get https://m.media-amazon.com/images/M/MV5BYTQxNTMwMmUtMWRkYi00MTRmLTgyYWItYTYwNGZkMWZmMzQ2XkEyXkFqcGdeQXVyMjM4NTM5NDY@._V1_SX300.jpg, data: [][0m
05-05 02:56:11 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/search/I+Trapped+the+Devil+2019/?limit=1, data: [][0m
05-05 02:56:11 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/search/movie?api_key=xxx&search_type=phrase&query=i+trapped+the+devil&year=2019, data: [][0m
05-05 02:56:11 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.omdbapi.com/?apikey=xxx&type=movie&y=2019&t=i+trapped+the+devil, data: [][0m
05-05 02:56:11 INFO [0m[ie.providers.info.omdbapi] Found: I Trapped the Devil (2019)[0m
05-05 02:56:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/537476?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:12 INFO [0m[providers.info.themoviedb] Found: [u'I Trapped the Devil (2019)'][0m
05-05 02:56:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/info/tt8708304/?ignore=1, data: [][0m
05-05 02:56:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.omdbapi.com/?apikey=xxx&type=movie&i=tt8708304, data: [][0m
05-05 02:56:12 INFO [0m[ie.providers.info.omdbapi] Found: I Trapped the Devil (2019)[0m
05-05 02:56:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt8708304?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/ismovie/tt8708304/, data: [][0m
05-05 02:56:12 INFO [0m[ie.providers.info.omdbapi] Found: I Trapped the Devil (2019)[0m
05-05 02:56:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt8708304?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:13 INFO [0m[ie.providers.info.omdbapi] Found: I Trapped the Devil (2019)[0m
05-05 02:56:13 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt8708304?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:13 INFO [0m[hpotato.core.plugins.base] Opening url: get https://m.media-amazon.com/images/M/MV5BNGNhMTQ0YWYtZDA5Mi00YWU2LWE4NzgtYTUzZGFhYTVkMzVlXkEyXkFqcGdeQXVyODY3Nzc0OTk@._V1_SX300.jpg, data: [][0m
05-05 02:56:20 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:56:21 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/isohunt2/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:56:23 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:56:24 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/search/The+Devils+Rejects+2005/?limit=1, data: [][0m
05-05 02:56:24 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/search/movie?api_key=xxx&search_type=phrase&query=the+devils+rejects&year=2005, data: [][0m
05-05 02:56:24 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.omdbapi.com/?apikey=xxx&type=movie&y=2005&t=the+devils+rejects, data: [][0m
05-05 02:56:24 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/1696?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:24 INFO [0m[providers.info.themoviedb] Found: [][0m
05-05 02:56:24 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/info/tt0395584/?ignore=1, data: [][0m
05-05 02:56:24 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.omdbapi.com/?apikey=xxx&type=movie&i=tt0395584, data: [][0m
05-05 02:56:25 INFO [0m[ie.providers.info.omdbapi] Found: The Devil's Rejects (2005)[0m
05-05 02:56:25 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt0395584?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:25 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/ismovie/tt0395584/, data: [][0m
05-05 02:56:25 INFO [0m[ie.providers.info.omdbapi] Found: The Devil's Rejects (2005)[0m
05-05 02:56:25 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt0395584?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:25 INFO [0m[ie.providers.info.omdbapi] Found: The Devil's Rejects (2005)[0m
05-05 02:56:25 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt0395584?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:25 INFO [0m[hpotato.core.plugins.base] Opening url: get https://m.media-amazon.com/images/M/MV5BMjIwMjM1MjM4NV5BMl5BanBnXkFtZTcwNTgyMzUzMw@@._V1_SX300.jpg, data: [][0m
05-05 02:56:33 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/search/Avengers+Endgame+2019/?limit=1, data: [][0m
05-05 02:56:33 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/search/movie?api_key=xxx&search_type=phrase&query=avengers+endgame&year=2019, data: [][0m
05-05 02:56:33 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.omdbapi.com/?apikey=xxx&type=movie&y=2019&t=avengers+endgame, data: [][0m
05-05 02:56:33 INFO [0m[ie.providers.info.omdbapi] Found: Avengers: Endgame (2019)[0m
05-05 02:56:33 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/299534?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:33 INFO [0m[providers.info.themoviedb] Found: [u'Avengers: Endgame (2019)'][0m
05-05 02:56:34 INFO [0m[ie.providers.info.omdbapi] Found: Avengers: Endgame (2019)[0m
05-05 02:56:34 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt4154796?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:34 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/ismovie/tt4154796/, data: [][0m
05-05 02:56:34 INFO [0m[ie.providers.info.omdbapi] Found: Avengers: Endgame (2019)[0m
05-05 02:56:34 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt4154796?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:34 INFO [0m[ie.providers.info.omdbapi] Found: Avengers: Endgame (2019)[0m
05-05 02:56:34 INFO [0m[hpotato.core.plugins.base] Opening url: get https://m.media-amazon.com/images/M/MV5BMTc5MDE2ODcwNV5BMl5BanBnXkFtZTgwMzI2NzQ2NzM@._V1_SX300.jpg, data: [][0m
05-05 02:56:35 INFO [0m[tato.core.plugins.scanner] Found 5 movies in the folder /media/nfsserver/Video/Movies[0m
05-05 02:56:53 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:56:53 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:56:56 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/ismovie/tt4263482/, data: [][0m
05-05 02:56:56 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/info/tt4263482/?ignore=1, data: [][0m
05-05 02:56:56 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt4263482?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:56 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.omdbapi.com/?apikey=xxx&type=movie&i=tt4263482, data: [][0m
05-05 02:56:57 INFO [0m[ie.providers.info.omdbapi] Found: The Witch (2015)[0m
05-05 02:56:57 INFO [0m[ie.providers.info.omdbapi] Found: The Witch (2015)[0m
05-05 02:56:57 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt4263482?api_key=xxx&append_to_response=alternative_titles&language=en, data: [][0m
05-05 02:56:57 INFO [0m[hpotato.core.plugins.base] Opening url: get https://m.media-amazon.com/images/M/MV5BMTUyNzkwMzAxOF5BMl5BanBnXkFtZTgwMzc1OTk1NjE@._V1_SX300.jpg, data: [][0m
05-05 02:56:57 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/eta/tt4263482/, data: [][0m
05-05 02:56:58 INFO [0m[core.media.movie.searcher] Search for The Witch in 720p ignoring ETA[0m
05-05 02:56:58 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.nzbgeek.info/api?imdbid=4263482&apikey=xxx&t=movie&extended=1&cat=2000, data: [][0m
05-05 02:56:58 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt4263482&adv_nfo=on&xminsize=3000&adv_age=1500&max=400&xmaxsize=10000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 02:57:02 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuNzIwcC5ibHVyYXkueDI2NC1kcm9uZXMtb2JmdXNjYXRlZA==/, data: [][0m
05-05 02:57:02 INFO [0m[edia._base.providers.base] Found: score(1754) on Newznab, api.nzbgeek.info: The.Witch.2015.720p.BluRay.x264-DRONES-Obfuscated[0m
05-05 02:57:03 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuNzIwcC5ibHVyYXkueDI2NC1kcm9uZXM=/, data: [][0m
05-05 02:57:04 INFO [0m[edia._base.providers.base] Found: score(1754) on Newznab, api.nzbgeek.info: The.Witch.2015.720p.BluRay.x264-DRONES[0m
05-05 02:57:04 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 02:57:04 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/gimmepeers/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:57:04 INFO [0m[hpotato.core.plugins.base] Opening url: get https://hd4free.xyz/searchapi.php?apikey=&username=&imdbid=tt4263482&internal=False, data: [][0m
05-05 02:57:05 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 02:57:05 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 02:57:05 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:57:09 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/1337x/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:57:14 INFO [0m[edia._base.providers.base] Found: score(1039) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[PRiME][1337x][0m
05-05 02:57:14 INFO [0m[edia._base.providers.base] Found: score(1052) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[EtHD][1337x][0m
05-05 02:57:15 INFO [0m[edia._base.providers.base] Found: score(1030) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.720p.BluRay.x264-EiDER[EtHD][1337x][0m
05-05 02:57:17 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:57:17 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:57:17 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/anidex/results/torznab/api?apikey=xxx&t=search&cat=&q=?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:57:23 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:57:23 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/limetorrents/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:57:25 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/magnetdl/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:57:27 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/nyaasi/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:57:27 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/z/zombieland-2-2019/se/desc/1/, data: [][0m
05-05 02:57:27 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Zombieland+2+2019%22/0/7/207, data: [][0m
05-05 02:57:27 INFO [0m[hpotato.core.plugins.base] Opening url: get https://yts.am/api/v2/list_movies.json?query_term=tt1560220&limit=10&page=1, data: [][0m
05-05 02:57:27 INFO [0m[hpotato.core.plugins.base] Opening url: get http://katcr.to/span/m-i1560220/, data: [][0m
05-05 02:57:29 INFO [0m[core.media.movie.searcher] Search for Zombieland 2 in 1080p ignoring ETA[0m
05-05 02:57:29 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt1560220&adv_nfo=on&xminsize=4000&adv_age=1500&max=400&xmaxsize=20000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 02:57:30 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 02:57:30 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:57:30 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:57:30 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:57:30 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:57:30 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:57:30 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 02:57:30 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 02:57:38 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:57:39 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/isohunt2/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:57:45 INFO [0m[edia._base.providers.base] Found: score(739) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[PRiME][Isohunt2][0m
05-05 02:57:45 INFO [0m[edia._base.providers.base] Found: score(752) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[EtHD][Isohunt2][0m
05-05 02:57:48 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:58:00 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:00 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:04 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/z/zombieland-2-2019/se/desc/1/, data: [][0m
05-05 02:58:04 INFO [0m[core.media.movie.searcher] Search for Zombieland 2 in BR-Rip ignoring ETA[0m
05-05 02:58:04 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt1560220&adv_nfo=on&xminsize=700&adv_age=1500&max=400&xmaxsize=7000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 02:58:05 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 02:58:05 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:05 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:58:05 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:05 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:58:05 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:05 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 02:58:05 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 02:58:16 INFO [0m[edia._base.providers.base] Found: score(3880) on TorrentPotato, 10.0.0.12: The Love Witch 2016 LiMiTED 720p BluRay x264-EiDER[EtHD][KATcrs][0m
05-05 02:58:17 INFO [0m[edia._base.providers.base] Found: score(3825) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER 720p BluRay x264-PSYCHD[EtHD][KATcrs][0m
05-05 02:58:17 INFO [0m[edia._base.providers.base] Found: score(2375) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER 720p BluRay x264-PSYCHD[PRiME][KATcrs][0m
05-05 02:58:18 INFO [0m[edia._base.providers.base] Found: score(4093) on TorrentPotato, 10.0.0.12: The Love Witch 2016 LiMiTED 720p BluRay x264-EiDER[rarbg][KATcrs][0m
05-05 02:58:18 INFO [0m[edia._base.providers.base] Found: score(2212) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER 720p BluRay x264-PSYCHD[rarbg][KATcrs][0m
05-05 02:58:18 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:58:24 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/limetorrents/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:58:27 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/magnetdl/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:58:29 INFO [0m[edia._base.providers.base] Found: score(345) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD [EtHD][MagnetDL][0m
05-05 02:58:29 INFO [0m[edia._base.providers.base] Found: score(345) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD [PRiME][MagnetDL][0m
05-05 02:58:29 INFO [0m[edia._base.providers.base] Found: score(330) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.720p.BluRay.x264-EiDER [EtHD][MagnetDL][0m
05-05 02:58:31 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/nyaasi/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:58:32 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/t/the-witch-2016/se/desc/1/, data: [][0m
05-05 02:58:32 INFO [0m[hpotato.core.plugins.base] Opening url: get https://yts.am/api/v2/list_movies.json?query_term=tt4263482&limit=10&page=1, data: [][0m
05-05 02:58:32 INFO [0m[hpotato.core.plugins.base] Opening url: get http://katcr.to/span/m-i4263482/, data: [][0m
05-05 02:58:32 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22The+Witch+2016%22/0/7/207, data: [][0m
05-05 02:58:33 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[PRiME][1337x][0m
05-05 02:58:33 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[PRiME][Isohunt2][0m
05-05 02:58:33 INFO [0m[tato.core.plugins.release] Tried to download, but none of the "torrent" downloaders are enabled or gave an error[0m
05-05 02:58:33 INFO [0m[core.media.movie.searcher] Search for The Witch in 1080p ignoring ETA[0m
05-05 02:58:33 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt4263482&adv_nfo=on&xminsize=4000&adv_age=1500&max=400&xmaxsize=20000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 02:58:33 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuMTA4MHAuYmx1cmF5LmR0cy54MjY0LWRyb25lcy1vYmZ1c2NhdGVk/, data: [][0m
05-05 02:58:34 INFO [0m[edia._base.providers.base] Found: score(1815) on Newznab, api.nzbgeek.info: The.Witch.2015.1080p.BluRay.DTS.x264-DRONES-Obfuscated[0m
05-05 02:58:34 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuMTA4MHAuYW16bi53ZWItZGw=/, data: [][0m
05-05 02:58:34 INFO [0m[edia._base.providers.base] Found: score(1800) on Newznab, api.nzbgeek.info: The.Witch.2015.1080p.AMZN.WEB-DL[0m
05-05 02:58:34 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUubXVsdGkuMTA4MHAuYmx1cmF5LngyNjQtcGlua3BhbnRlcnMtb2JmdXNjYXRlZA==/, data: [][0m
05-05 02:58:34 INFO [0m[edia._base.providers.base] Found: score(1811) on Newznab, api.nzbgeek.info: the.witch.2015.multi.1080p.bluray.x264-pinkpanters-Obfuscated[0m
05-05 02:58:35 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUubXVsdGkuMTA4MHAuYmx1cmF5LngyNjQtcGlua3BhbnRlcnM=/, data: [][0m
05-05 02:58:35 INFO [0m[edia._base.providers.base] Found: score(1811) on Newznab, api.nzbgeek.info: the.witch.2015.multi.1080p.bluray.x264-pinkpanters[0m
05-05 02:58:35 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuMTA4MHAuYmx1cmF5LngyNjQtZHJvbmVzLW9iZnVzY2F0ZWQ=/, data: [][0m
05-05 02:58:35 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:35 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Zombieland+2+2019%22/0/7/201,207, data: [][0m
05-05 02:58:35 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/z/zombieland-2-2019/se/desc/1/, data: [][0m
05-05 02:58:35 INFO [0m[edia._base.providers.base] Found: score(1811) on Newznab, api.nzbgeek.info: The.Witch.2015.1080p.BluRay.x264-DRONES-Obfuscated[0m
05-05 02:58:36 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuMTA4MHAuYmx1cmF5LmR0cy54MjY0LWN5dHN1bmVlLW9iZnVzY2F0ZWQ=/, data: [][0m
05-05 02:58:36 INFO [0m[edia._base.providers.base] Found: score(1815) on Newznab, api.nzbgeek.info: The.Witch.2015.1080p.BluRay.DTS.x264-CyTSuNee-Obfuscated[0m
05-05 02:58:36 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuMTA4MHAuYmx1cmF5LmR0cy54MjY0LWN5dHN1bmVl/, data: [][0m
05-05 02:58:36 INFO [0m[edia._base.providers.base] Found: score(1815) on Newznab, api.nzbgeek.info: The.Witch.2015.1080p.BluRay.DTS.x264-CyTSuNee[0m
05-05 02:58:36 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuMTA4MHAuYmx1cmF5LngyNjQtZGl2dWxnZWQ=/, data: [][0m
05-05 02:58:36 INFO [0m[edia._base.providers.base] Found: score(1811) on Newznab, api.nzbgeek.info: The.Witch.2015.1080p.BluRay.x264-DiVULGED[0m
05-05 02:58:37 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLnZ2aXRjaC5hLm5ldy1lbmdsYW5kLmZvbGt0YWxlLjIwMTUuMTA4MHAuYmx1cmF5LmRkNS4xLngyNjQtdmlzdW0=/, data: [][0m
05-05 02:58:37 INFO [0m[edia._base.providers.base] Found: score(1663) on Newznab, api.nzbgeek.info: The.VVitch.A.New-England.Folktale.2015.1080p.BluRay.DD5.1.x264-ViSUM[0m
05-05 02:58:38 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuMTA4MHAuYmx1cmF5LngyNjQtZHJvbmVz/, data: [][0m
05-05 02:58:38 INFO [0m[edia._base.providers.base] Found: score(1811) on Newznab, api.nzbgeek.info: The.Witch.2015.1080p.BluRay.x264-DRONES[0m
05-05 02:58:38 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 02:58:38 INFO [0m[hpotato.core.plugins.base] Opening url: get https://hd4free.xyz/searchapi.php?apikey=&username=&imdbid=tt4263482&internal=False, data: [][0m
05-05 02:58:38 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/1337x/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:58:38 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 02:58:38 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 02:58:39 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:58:42 INFO [0m[edia._base.providers.base] Found: score(1048) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.BluRay.x264-PSYCHD[1337x][0m
05-05 02:58:42 INFO [0m[edia._base.providers.base] Found: score(1048) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.1080p.BluRay.x264-EiDER[EtHD][1337x][0m
05-05 02:58:42 INFO [0m[hpotato.core.plugins.base] Opening url: get https://yts.am/api/v2/list_movies.json?query_term=tt1560220&limit=10&page=1, data: [][0m
05-05 02:58:43 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:58:43 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:58:43 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/anidex/results/torznab/api?apikey=xxx&t=search&cat=&q=?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:58:43 INFO [0m[core.media.movie.searcher] Search for Zombieland 2 in BR-Disk ignoring ETA[0m
05-05 02:58:43 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt1560220&adv_nfo=on&xminsize=20000&adv_age=1500&max=400&xmaxsize=60000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 02:58:44 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 02:58:44 INFO [0m[hpotato.core.plugins.base] Opening url: get https://hd4free.xyz/searchapi.php?apikey=&username=&imdbid=tt1560220&internal=False, data: [][0m
05-05 02:58:44 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/gimmepeers/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:44 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 02:58:44 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 02:58:44 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:58:49 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/1337x/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:52 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:52 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:52 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:58:53 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:58:58 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:58:59 INFO [0m[edia._base.providers.base] Found: score(748) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.BluRay.x264-PSYCHD[Isohunt2][0m
05-05 02:58:59 INFO [0m[edia._base.providers.base] Found: score(748) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.1080p.BluRay.x264-EiDER[EtHD][Isohunt2][0m
05-05 02:59:02 INFO [0m[edia._base.providers.base] Found: score(3671) on TorrentPotato, 10.0.0.12: The Love Witch 2016 LiMiTED 1080p BluRay x264-EiDER[EtHD][KATcrs][0m
05-05 02:59:02 INFO [0m[edia._base.providers.base] Found: score(6722) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER 1080p BluRay x264-PSYCHD[rarbg][KATcrs][0m
05-05 02:59:02 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:59:05 INFO [0m[edia._base.providers.base] Found: score(354) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.BluRay.x264-PSYCHD[MagnetDL][0m
05-05 02:59:05 INFO [0m[edia._base.providers.base] Found: score(365) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.Bluray.1080p.DTS-HD.x264-Grym[MagnetDL][0m
05-05 02:59:05 INFO [0m[edia._base.providers.base] Found: score(354) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.1080p.BluRay.x264-EiDER [EtHD][MagnetDL][0m
05-05 02:59:06 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/t/the-witch-2016/se/desc/1/, data: [][0m
05-05 02:59:06 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.1080p.BluRay.x264-PSYCHD[1337x][0m
05-05 02:59:06 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.LiMiTED.1080p.BluRay.x264-EiDER[EtHD][1337x][0m
05-05 02:59:06 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.1080p.BluRay.x264-PSYCHD[Isohunt2][0m
05-05 02:59:06 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.LiMiTED.1080p.BluRay.x264-EiDER[EtHD][Isohunt2][0m
05-05 02:59:06 INFO [0m[tato.core.plugins.release] Tried to download, but none of the "torrent" downloaders are enabled or gave an error[0m
05-05 02:59:06 INFO [0m[core.media.movie.searcher] Search for The Witch in BR-Rip ignoring ETA[0m
05-05 02:59:06 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt4263482&adv_nfo=on&xminsize=700&adv_age=1500&max=400&xmaxsize=7000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 02:59:07 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuaGRyaXAueHZpZC5hYzMtZXZv/, data: [][0m
05-05 02:59:07 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO[0m
05-05 02:59:07 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO[0m
05-05 02:59:07 INFO [0m[edia._base.providers.base] Found: score(127) on BinSearch: 1 20 description 20160367 nfo 2 40 gb yenc 1 1[0m
05-05 02:59:07 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO[0m
05-05 02:59:08 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO[0m
05-05 02:59:08 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO[0m
05-05 02:59:08 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO[0m
05-05 02:59:08 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuMTA4MHAuYW16bi53ZWItZGwuZGQrNS4xLngyNjQtbW9ua2Vl/, data: [][0m
05-05 02:59:08 INFO [0m[edia._base.providers.base] Found: score(1858) on Newznab, api.nzbgeek.info: The.Witch.2015.1080p.AMZN.WEB-DL.DD+5.1.x264-monkee[0m
05-05 02:59:09 INFO [0m[edia._base.providers.base] Found: score(236) on BinSearch: the witch 2015 720p internal bluray x264 bwhj nfo 01 68 the witch 2015 720p internal bluray x264 bwhj nfo yenc 1 1[0m
05-05 02:59:09 INFO [0m[edia._base.providers.base] Found: score(1857) on Newznab, api.nzbgeek.info: The.Witch.2015.1080p.AMZN.WEB-DL[0m
05-05 02:59:09 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO[0m
05-05 02:59:09 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO[0m
05-05 02:59:09 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuMTA4MHAuYmx1cmF5LngyNjUuaW50ZXJuYWwtZmxhbWU=/, data: [][0m
05-05 02:59:09 INFO [0m[edia._base.providers.base] Found: score(1867) on Newznab, api.nzbgeek.info: The.Witch.2015.1080p.BluRay.x265.INTERNAL-FLAME[0m
05-05 02:59:09 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO[0m
05-05 02:59:10 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO[0m
05-05 02:59:10 INFO [0m[edia._base.providers.base] Found: score(1870) on Newznab, api.nzbgeek.info: The Witch 2015 720p Bluray AC3 x264 - AdiT[0m
05-05 02:59:10 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuNzIwcC5ibHVyYXkuYWMzLngyNjQtYWRpdA==/, data: [][0m
05-05 02:59:10 INFO [0m[edia._base.providers.base] Found: score(1870) on Newznab, api.nzbgeek.info: The.Witch.2015.720p.Bluray.AC3.x264-AdiT[0m
05-05 02:59:10 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuNzIwcC53ZWItZGwuaDI2NC5hYzMtZXZvLW9iZnVzY2F0ZWQ=/, data: [][0m
05-05 02:59:11 INFO [0m[edia._base.providers.base] Found: score(1860) on Newznab, api.nzbgeek.info: The.Witch.2015.720p.WEB-DL.H264.AC3-EVO-Obfuscated[0m
05-05 02:59:11 INFO [0m[edia._base.providers.base] Found: score(1868) on Newznab, api.nzbgeek.info: The.Witch.2015.720p.BluRay.x264-DRONES-Obfuscated[0m
05-05 02:59:11 INFO [0m[edia._base.providers.base] Found: score(1848) on Newznab, api.nzbgeek.info: The.Witch.2015.Deutsch.BDRip.LD.XViD by Admin f?r GF[0m
05-05 02:59:11 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuYmx1cmF5LjcyMHAuaDI2NC0yMC00MA==/, data: [][0m
05-05 02:59:11 INFO [0m[edia._base.providers.base] Found: score(1948) on Newznab, api.nzbgeek.info: The.Witch.2015.BluRay.720p.H264-20-40[0m
05-05 02:59:11 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuaGRyaXAueHZpZC5hYzMtZXZvLW9iZnVzY2F0ZWQ=/, data: [][0m
05-05 02:59:12 INFO [0m[edia._base.providers.base] Found: score(1849) on Newznab, api.nzbgeek.info: The.Witch.2015.HDRip.XviD.AC3-EVO-Obfuscated[0m
05-05 02:59:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuYnJyaXAueHZpZC5hYzMtZXZvLW9iZnVzY2F0ZWQ=/, data: [][0m
05-05 02:59:12 INFO [0m[edia._base.providers.base] Found: score(1850) on Newznab, api.nzbgeek.info: The.Witch.2015.BRRip.XviD.AC3-EVO-Obfuscated[0m
05-05 02:59:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuNzIwcC5ibHVyYXkuaDI2NC5hYWMtcmFyYmctb2JmdXNjYXRlZA==/, data: [][0m
05-05 02:59:12 INFO [0m[edia._base.providers.base] Found: score(1868) on Newznab, api.nzbgeek.info: The.Witch.2015.720p.BluRay.H264.AAC-RARBG-Obfuscated[0m
05-05 02:59:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuNzIwcC5icnJpcC5hYWMuMi4wLngyNjQtc3Jn/, data: [][0m
05-05 02:59:12 INFO [0m[edia._base.providers.base] Found: score(1939) on Newznab, api.nzbgeek.info: The.Witch.2015.720p.BRRip.AAC.2.0.x264-SRG[0m
05-05 02:59:13 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuNzIwcC5icnJpcC54dmlkLmFjMy1yYXJiZy1vYmZ1c2NhdGVk/, data: [][0m
05-05 02:59:13 INFO [0m[edia._base.providers.base] Found: score(1940) on Newznab, api.nzbgeek.info: The.Witch.2015.720p.BRRip.XviD.AC3-RARBG-Obfuscated[0m
05-05 02:59:13 INFO [0m[edia._base.providers.base] Found: score(1868) on Newznab, api.nzbgeek.info: The.Witch.2015.720p.BluRay.x264-DRONES[0m
05-05 02:59:13 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuNzIwcC5oZHJpcC54MjY0LmFjMy10aXRhbi1jaGFtZWxlMG4=/, data: [][0m
05-05 02:59:13 INFO [0m[edia._base.providers.base] Found: score(1940) on Newznab, api.nzbgeek.info: The.Witch.2015.720p.HDRip.x264.AC3-TiTAN-Chamele0n[0m
05-05 02:59:13 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 02:59:13 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:59:13 INFO [0m[edia._base.providers.base] Found: score(1072) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BluRay.H264.AAC[1337x][0m
05-05 02:59:14 INFO [0m[edia._base.providers.base] Found: score(1078) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.BluRay.H264.AAC[1337x][0m
05-05 02:59:14 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 02:59:14 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 02:59:14 INFO [0m[edia._base.providers.base] Found: score(1071) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.MP3-XVID[1337x][0m
05-05 02:59:14 INFO [0m[edia._base.providers.base] Found: score(1078) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BluRay.x264-x0r[PRiME][1337x][0m
05-05 02:59:14 INFO [0m[edia._base.providers.base] Found: score(1064) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.ENG.Sub.ITA.HDRip.XviD-[WEB][1337x][0m
05-05 02:59:14 INFO [0m[edia._base.providers.base] Found: score(1339) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BRRip.x264.AAC-ETRG[1337x][0m
05-05 02:59:14 INFO [0m[edia._base.providers.base] Found: score(1250) on TorrentPotato, 10.0.0.12: The Love Witch (2016) [1080p] [YTS] [YIFY][1337x][0m
05-05 02:59:14 INFO [0m[edia._base.providers.base] Found: score(1264) on TorrentPotato, 10.0.0.12: The Love Witch (2016) [720p] [YTS] [YIFY][1337x][0m
05-05 02:59:15 INFO [0m[edia._base.providers.base] Found: score(1018) on TorrentPotato, 10.0.0.12: 18+ The Love Witch 2016 UNCENSORED Movies 720p BluRay x264 AAC New Source with Sample ☻rDX☻[1337x][0m
05-05 02:59:15 INFO [0m[edia._base.providers.base] Found: score(1087) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[PRiME][1337x][0m
05-05 02:59:15 INFO [0m[edia._base.providers.base] Found: score(1100) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[EtHD][1337x][0m
05-05 02:59:15 INFO [0m[edia._base.providers.base] Found: score(1077) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD[PRiME][1337x][0m
05-05 02:59:15 INFO [0m[edia._base.providers.base] Found: score(1074) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD[1337x][0m
05-05 02:59:15 INFO [0m[edia._base.providers.base] Found: score(1060) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[1337x][0m
05-05 02:59:15 INFO [0m[edia._base.providers.base] Found: score(1060) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[PRiME][1337x][0m
05-05 02:59:15 INFO [0m[edia._base.providers.base] Found: score(1060) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[1337x][0m
05-05 02:59:16 INFO [0m[edia._base.providers.base] Found: score(1053) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.BDRip.x264-EiDER[1337x][0m
05-05 02:59:16 INFO [0m[edia._base.providers.base] Found: score(1059) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.BDRip.x264-EiDER[PRiME][1337x][0m
05-05 02:59:16 INFO [0m[edia._base.providers.base] Found: score(1078) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.720p.BluRay.x264-EiDER[EtHD][1337x][0m
05-05 02:59:16 INFO [0m[edia._base.providers.base] Found: score(1065) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.DD2.0.x264-BDP[1337x][0m
05-05 02:59:16 INFO [0m[edia._base.providers.base] Found: score(1058) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.DD2.0.x264-BDP[PRiME][1337x][0m
05-05 02:59:16 INFO [0m[edia._base.providers.base] Found: score(1068) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.WEB-DL.DD5.1.x264-BDP[PRiME][1337x][0m
05-05 02:59:16 INFO [0m[edia._base.providers.base] Found: score(1062) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.WEB-DL.x264 - WeTv[1337x][0m
05-05 02:59:16 INFO [0m[edia._base.providers.base] Found: score(998) on TorrentPotato, 10.0.0.12: 18+ The Love Witch 2016 UNCENSORED Movies 720p HDRip XviD ESubs AAC New +Sample ☻rDX☻[1337x][0m
05-05 02:59:17 INFO [0m[edia._base.providers.base] Found: score(1062) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.WEB-DL.x264 - WeTv[1337x][0m
05-05 02:59:17 INFO [0m[edia._base.providers.base] Found: score(1053) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[1337x][0m
05-05 02:59:17 INFO [0m[edia._base.providers.base] Found: score(1065) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-iFT[1337x][0m
05-05 02:59:17 INFO [0m[edia._base.providers.base] Found: score(1059) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-iFT[PRiME][1337x][0m
05-05 02:59:17 INFO [0m[edia._base.providers.base] Found: score(1163) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[1337x][0m
05-05 02:59:17 INFO [0m[edia._base.providers.base] Found: score(1128) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.WEB-DL.DD5.1.H264-FGT[1337x][0m
05-05 02:59:17 INFO [0m[edia._base.providers.base] Found: score(1063) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.WEB-DL.XviD.AC3-FGT[1337x][0m
05-05 02:59:17 INFO [0m[edia._base.providers.base] Found: score(1090) on TorrentPotato, 10.0.0.12: The Love Witch (2016) 720p WEB-DL 950MB - MkvCage[1337x][0m
05-05 02:59:17 INFO [0m[edia._base.providers.base] Found: score(1091) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.WEB-DL.DD5.1.H264-FGT[EtHD][1337x][0m
05-05 02:59:18 INFO [0m[edia._base.providers.base] Found: score(1059) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[1337x][0m
05-05 02:59:18 INFO [0m[edia._base.providers.base] Found: score(1053) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[PRiME][1337x][0m
05-05 02:59:18 INFO [0m[edia._base.providers.base] Found: score(1222) on TorrentPotato, 10.0.0.12: The.Witch.2016.720p.BluRay.x264-NeZu[1337x][0m
05-05 02:59:18 INFO [0m[edia._base.providers.base] Found: score(1249) on TorrentPotato, 10.0.0.12: The Witch 2016 1080p HDRip x264 AAC-m2g[1337x][0m
05-05 02:59:18 INFO [0m[edia._base.providers.base] Found: score(1251) on TorrentPotato, 10.0.0.12: The Witch 2016 1080p HDRip x264 AC3 6CH-CPG[1337x][0m
05-05 02:59:18 INFO [0m[edia._base.providers.base] Found: score(1155) on TorrentPotato, 10.0.0.12: The Witch 2016 HDRip x264 AC3 6CH-CPG[1337x][0m
05-05 02:59:18 INFO [0m[edia._base.providers.base] Found: score(887) on TorrentPotato, 10.0.0.12: 18+ Mark Of The Witch 2016 English Movies HDRip XviD AAC New Source with Sample ~ ☻rDX☻[1337x][0m
05-05 02:59:19 INFO [0m[edia._base.providers.base] Found: score(1081) on TorrentPotato, 10.0.0.12: Mark.of.the.Witch.2016.HDRip.XviD.AC3-EVO[1337x][0m
05-05 02:59:19 INFO [0m[edia._base.providers.base] Found: score(1038) on TorrentPotato, 10.0.0.12: Mark of the Witch 2016 HDRip XviD AC3-EVO[1337x][0m
05-05 02:59:19 INFO [0m[edia._base.providers.base] Found: score(1032) on TorrentPotato, 10.0.0.12: Mark of the Witch 2016 HDRip XviD AC3-EVO[1337x][0m
05-05 02:59:19 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:59:19 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:59:19 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:59:19 INFO [0m[edia._base.providers.base] Found: score(772) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BluRay.H264.AAC[Isohunt2][0m
05-05 02:59:19 INFO [0m[edia._base.providers.base] Found: score(778) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.BluRay.H264.AAC[Isohunt2][0m
05-05 02:59:19 INFO [0m[edia._base.providers.base] Found: score(768) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.MP3-XVID[Isohunt2][0m
05-05 02:59:19 INFO [0m[edia._base.providers.base] Found: score(778) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BluRay.x264-x0r[PRiME][Isohunt2][0m
05-05 02:59:20 INFO [0m[edia._base.providers.base] Found: score(764) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.ENG.Sub.ITA.HDRip.XviD-[WEB][Isohunt2][0m
05-05 02:59:20 INFO [0m[edia._base.providers.base] Found: score(1009) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BRRip.x264.AAC-ETRG[Isohunt2][0m
05-05 02:59:20 INFO [0m[edia._base.providers.base] Found: score(927) on TorrentPotato, 10.0.0.12: The Love Witch (2016) [1080p] [YTS] [YIFY][Isohunt2][0m
05-05 02:59:20 INFO [0m[edia._base.providers.base] Found: score(954) on TorrentPotato, 10.0.0.12: The Love Witch (2016) [720p] [YTS] [YIFY][Isohunt2][0m
05-05 02:59:20 INFO [0m[edia._base.providers.base] Found: score(715) on TorrentPotato, 10.0.0.12: 18+ The Love Witch 2016 UNCENSORED Movies 720p BluRay x264 AAC Ne...[Isohunt2][0m
05-05 02:59:20 INFO [0m[edia._base.providers.base] Found: score(787) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[PRiME][Isohunt2][0m
05-05 02:59:20 INFO [0m[edia._base.providers.base] Found: score(800) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[EtHD][Isohunt2][0m
05-05 02:59:20 INFO [0m[edia._base.providers.base] Found: score(774) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD[PRiME][Isohunt2][0m
05-05 02:59:21 INFO [0m[edia._base.providers.base] Found: score(774) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD[Isohunt2][0m
05-05 02:59:21 INFO [0m[edia._base.providers.base] Found: score(760) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:21 INFO [0m[edia._base.providers.base] Found: score(760) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[PRiME][Isohunt2][0m
05-05 02:59:21 INFO [0m[edia._base.providers.base] Found: score(760) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:21 INFO [0m[edia._base.providers.base] Found: score(768) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD[Isohunt2][0m
05-05 02:59:21 INFO [0m[edia._base.providers.base] Found: score(763) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BRRip.x264.AAC-ETRG[Isohunt2][0m
05-05 02:59:21 INFO [0m[edia._base.providers.base] Found: score(754) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:21 INFO [0m[edia._base.providers.base] Found: score(753) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.BDRip.x264-EiDER[Isohunt2][0m
05-05 02:59:22 INFO [0m[edia._base.providers.base] Found: score(758) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.DD2.0.x264-BDP[Isohunt2][0m
05-05 02:59:22 INFO [0m[edia._base.providers.base] Found: score(704) on TorrentPotato, 10.0.0.12: 18+ The Love Witch 2016 UNCENSORED Movies 720p HDRip XviD ESubs[Isohunt2][0m
05-05 02:59:22 INFO [0m[edia._base.providers.base] Found: score(753) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-iFT[Isohunt2][0m
05-05 02:59:22 INFO [0m[edia._base.providers.base] Found: score(786) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:22 INFO [0m[edia._base.providers.base] Found: score(753) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:22 INFO [0m[edia._base.providers.base] Found: score(752) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.DD2.0.x264-BDP[PRiME][Isohunt2][0m
05-05 02:59:22 INFO [0m[edia._base.providers.base] Found: score(768) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.WEB-DL.DD5.1.x264-BDP[PRiME][Isohunt2][0m
05-05 02:59:22 INFO [0m[edia._base.providers.base] Found: score(762) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.WEB-DL.x264 - WeTv[Isohunt2][0m
05-05 02:59:23 INFO [0m[edia._base.providers.base] Found: score(718) on TorrentPotato, 10.0.0.12: 18+ The Love Witch 2016 UNCENSORED Movies 720p HDRip XviD ESubs A...[Isohunt2][0m
05-05 02:59:23 INFO [0m[edia._base.providers.base] Found: score(775) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.WEB-DL.x264 - WeTv[Isohunt2][0m
05-05 02:59:23 INFO [0m[edia._base.providers.base] Found: score(753) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:23 INFO [0m[edia._base.providers.base] Found: score(753) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-iFT[Isohunt2][0m
05-05 02:59:23 INFO [0m[edia._base.providers.base] Found: score(766) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-iFT[PRiME][Isohunt2][0m
05-05 02:59:23 INFO [0m[edia._base.providers.base] Found: score(853) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:23 INFO [0m[edia._base.providers.base] Found: score(855) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.WEB-DL.DD5.1.H264-FGT[Isohunt2][0m
05-05 02:59:23 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:59:23 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/z/zombieland-2-2019/se/desc/1/, data: [][0m
05-05 02:59:24 INFO [0m[edia._base.providers.base] Found: score(763) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.WEB-DL.XviD.AC3-FGT[Isohunt2][0m
05-05 02:59:24 INFO [0m[edia._base.providers.base] Found: score(794) on TorrentPotato, 10.0.0.12: The Love Witch (2016) 720p WEB-DL 950MB - MkvCage[Isohunt2][0m
05-05 02:59:24 INFO [0m[edia._base.providers.base] Found: score(788) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.WEB-DL.DD5.1.H264-FGT[EtHD][Isohunt2][0m
05-05 02:59:24 INFO [0m[edia._base.providers.base] Found: score(753) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:24 INFO [0m[edia._base.providers.base] Found: score(753) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[PRiME][Isohunt2][0m
05-05 02:59:24 INFO [0m[core.media.movie.searcher] Search for Zombieland 2 in DVD-Rip ignoring ETA[0m
05-05 02:59:25 INFO [0m[edia._base.providers.base] Found: score(919) on TorrentPotato, 10.0.0.12: The.Witch.2016.720p.BluRay.x264-NeZu[Isohunt2][0m
05-05 02:59:25 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt1560220&adv_nfo=on&xminsize=600&adv_age=1500&max=400&xmaxsize=2400&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 02:59:25 INFO [0m[edia._base.providers.base] Found: score(694) on TorrentPotato, 10.0.0.12: 18+ Mark Of The Witch 2016 Movies HDRip XviD New +Sample ~ ☻rDX☻[Isohunt2][0m
05-05 02:59:25 INFO [0m[edia._base.providers.base] Found: score(738) on TorrentPotato, 10.0.0.12: Mark of the Witch 2016 HDRip XviD AC3-EVO[Isohunt2][0m
05-05 02:59:25 INFO [0m[edia._base.providers.base] Found: score(805) on TorrentPotato, 10.0.0.12: Mark.of.the.Witch.2016.HDRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:25 INFO [0m[edia._base.providers.base] Found: score(732) on TorrentPotato, 10.0.0.12: Mark of the Witch 2016 HDRip XviD AC3-EVO[Isohunt2][0m
05-05 02:59:25 INFO [0m[edia._base.providers.base] Found: score(1988) on TorrentPotato, 10.0.0.12: The Love Witch 2016 1080p WEB-DL DD5 1 H264-FGT[KATcrs][0m
05-05 02:59:26 INFO [0m[edia._base.providers.base] Found: score(4185) on TorrentPotato, 10.0.0.12: The Love Witch 2016 HDRip XviD AC3-EVO[KATcrs][0m
05-05 02:59:26 INFO [0m[edia._base.providers.base] Found: score(4332) on TorrentPotato, 10.0.0.12: The Love Witch 2016 HDRip XviD AC3-EVO[PRiME][KATcrs][0m
05-05 02:59:26 INFO [0m[edia._base.providers.base] Found: score(5851) on TorrentPotato, 10.0.0.12: The Love Witch 2016 1080p WEB-DL DD5 1 H264-FGT[EtHD][KATcrs][0m
05-05 02:59:26 INFO [0m[edia._base.providers.base] Found: score(4849) on TorrentPotato, 10.0.0.12: The Love Witch 2016 HDRip XviD AC3-EVO[SN][KATcrs][0m
05-05 02:59:26 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 02:59:26 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:59:26 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:59:26 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:59:26 INFO [0m[edia._base.providers.base] Found: score(5090) on TorrentPotato, 10.0.0.12: The Love Witch (2016) 720p WEB-DL 950MB - MkvCage[KATcrs][0m
05-05 02:59:26 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:59:26 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:59:26 INFO [0m[edia._base.providers.base] Found: score(6032) on TorrentPotato, 10.0.0.12: The Love Witch 2016 720p WEB-DL XviD AC3-FGT[KATcrs][0m
05-05 02:59:26 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 02:59:26 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 02:59:27 INFO [0m[edia._base.providers.base] Found: score(5785) on TorrentPotato, 10.0.0.12: The Love Witch 2016 HDRip XviD AC3-iFT[PRiME][KATcrs][0m
05-05 02:59:27 INFO [0m[edia._base.providers.base] Found: score(5722) on TorrentPotato, 10.0.0.12: The Love Witch 2016 HDRip XviD AC3-iFT[KATcrs][0m
05-05 02:59:27 INFO [0m[edia._base.providers.base] Found: score(2311) on TorrentPotato, 10.0.0.12: The Love Witch 2016 720p WEB-DL 950MB MkvCage mkv[KATcrs][0m
05-05 02:59:27 INFO [0m[edia._base.providers.base] Found: score(4613) on TorrentPotato, 10.0.0.12: The Love Witch 2016 HDRip XviD AC3-iFT[SN][KATcrs][0m
05-05 02:59:27 INFO [0m[edia._base.providers.base] Found: score(5488) on TorrentPotato, 10.0.0.12: The love witch 2016 720p web-dl x264-NBY-[moviezplanet org] mkv[KATcrs][0m
05-05 02:59:27 INFO [0m[edia._base.providers.base] Found: score(6454) on TorrentPotato, 10.0.0.12: The Love Witch 2016 1080p WEB-DL x264 - WeTv[KATcrs][0m
05-05 02:59:27 INFO [0m[edia._base.providers.base] Found: score(962) on TorrentPotato, 10.0.0.12: The Love Witch 2016 720p WEB-DL x264 ShAaNiG mkv[KATcrs][0m
05-05 02:59:27 INFO [0m[edia._base.providers.base] Found: score(2251) on TorrentPotato, 10.0.0.12: 18+ The Love Witch 2016 UNCENSORED Movies 720p HDRip XviD ESubs AAC New +Sample ☻rDX☻[KATcrs][0m
05-05 02:59:28 INFO [0m[edia._base.providers.base] Found: score(1888) on TorrentPotato, 10.0.0.12: The Love Witch 2016 720p WEB-DL x264 - WeTv[KATcrs][0m
05-05 02:59:28 INFO [0m[edia._base.providers.base] Found: score(3274) on TorrentPotato, 10.0.0.12: The Love Witch 2016 720p WEB-DL DD5 1 x264-BDP[PRiME][KATcrs][0m
05-05 02:59:28 INFO [0m[edia._base.providers.base] Found: score(2398) on TorrentPotato, 10.0.0.12: The Love Witch 2016 HDRip DD2 0 x264-BDP[PRiME][KATcrs][0m
05-05 02:59:28 INFO [0m[edia._base.providers.base] Found: score(4751) on TorrentPotato, 10.0.0.12: The Love Witch 2016 HDRip DD2 0 x264-BDP[KATcrs][0m
05-05 02:59:28 INFO [0m[edia._base.providers.base] Found: score(5745) on TorrentPotato, 10.0.0.12: The Love Witch 2016 HDRip DD2 0 x264-BDP[SN][KATcrs][0m
05-05 02:59:28 INFO [0m[edia._base.providers.base] Found: score(6069) on TorrentPotato, 10.0.0.12: The Love Witch 2016 LiMiTED BDRip x264-EiDER[KATcrs][0m
05-05 02:59:28 INFO [0m[edia._base.providers.base] Found: score(6145) on TorrentPotato, 10.0.0.12: The Love Witch 2016 LiMiTED BDRip x264-EiDER[PRiME][KATcrs][0m
05-05 02:59:29 INFO [0m[edia._base.providers.base] Found: score(3928) on TorrentPotato, 10.0.0.12: The Love Witch 2016 LiMiTED 720p BluRay x264-EiDER[EtHD][KATcrs][0m
05-05 02:59:29 INFO [0m[edia._base.providers.base] Found: score(5726) on TorrentPotato, 10.0.0.12: The Love Witch 2016 LiMiTED BDRip x264-EiDER[1337x][SN][KATcrs][0m
05-05 02:59:29 INFO [0m[edia._base.providers.base] Found: score(1950) on TorrentPotato, 10.0.0.12: The Love Witch 2016 BRRip XviD AC3-EVO[PRiME][KATcrs][0m
05-05 02:59:29 INFO [0m[edia._base.providers.base] Found: score(3707) on TorrentPotato, 10.0.0.12: The Love Witch 2016 BRRip XviD AC3-EVO[KATcrs][0m
05-05 02:59:29 INFO [0m[edia._base.providers.base] Found: score(4210) on TorrentPotato, 10.0.0.12: The Love Witch 2016 BRRip XviD AC3-EVO[SN][KATcrs][0m
05-05 02:59:29 INFO [0m[edia._base.providers.base] Found: score(5303) on TorrentPotato, 10.0.0.12: The Love Witch 2016 720p BRRip x264 AAC-ETRG[KATcrs][0m
05-05 02:59:29 INFO [0m[edia._base.providers.base] Found: score(2900) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER BDRip x264-PSYCHD[PRiME][KATcrs][0m
05-05 02:59:29 INFO [0m[edia._base.providers.base] Found: score(5630) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER BDRip x264-PSYCHD[KATcrs][0m
05-05 02:59:30 INFO [0m[edia._base.providers.base] Found: score(3873) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER 720p BluRay x264-PSYCHD[EtHD][KATcrs][0m
05-05 02:59:30 INFO [0m[edia._base.providers.base] Found: score(2423) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER 720p BluRay x264-PSYCHD[PRiME][KATcrs][0m
05-05 02:59:30 INFO [0m[edia._base.providers.base] Found: score(6214) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER BDRip x264-PSYCHD[1337x][SN][KATcrs][0m
05-05 02:59:30 INFO [0m[edia._base.providers.base] Found: score(2276) on TorrentPotato, 10.0.0.12: The love witch 2016 proper 720p bluray x264-NBY-[[KATcrs][0m
05-05 02:59:30 INFO [0m[edia._base.providers.base] Found: score(4175) on TorrentPotato, 10.0.0.12: 18+ The Love Witch 2016 UNCENSORED Movies 720p BluRay x264 AAC New Source with Sample ☻rDX☻[KATcrs][0m
05-05 02:59:30 INFO [0m[edia._base.providers.base] Found: score(4328) on TorrentPotato, 10.0.0.12: The Love Witch 2016 720p BRRip 1GB MkvCage mkv[KATcrs][0m
05-05 02:59:30 INFO [0m[edia._base.providers.base] Found: score(3198) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER BDRip x264-PSYCHD[rarbg][KATcrs][0m
05-05 02:59:30 INFO [0m[edia._base.providers.base] Found: score(2207) on TorrentPotato, 10.0.0.12: The Love Witch 2016 BRRip XviD AC3-RARBG[KATcrs][0m
05-05 02:59:31 INFO [0m[edia._base.providers.base] Found: score(4141) on TorrentPotato, 10.0.0.12: The Love Witch 2016 LiMiTED 720p BluRay x264-EiDER[rarbg][KATcrs][0m
05-05 02:59:31 INFO [0m[edia._base.providers.base] Found: score(2260) on TorrentPotato, 10.0.0.12: The Love Witch 2016 PROPER 720p BluRay x264-PSYCHD[rarbg][KATcrs][0m
05-05 02:59:31 INFO [0m[edia._base.providers.base] Found: score(532) on TorrentPotato, 10.0.0.12: Mark of the Witch 2016 HDRip XviD AC3 EVO[KickAssTorrent][0m
05-05 02:59:31 INFO [0m[edia._base.providers.base] Found: score(662) on TorrentPotato, 10.0.0.12: The Witch 2016 (1080p x265 10bit Joy)[KickAssTorrent][0m
05-05 02:59:31 INFO [0m[edia._base.providers.base] Found: score(65) on TorrentPotato, 10.0.0.12: Czarownica: Bajka ludowa z Nowej Anglii - The Witch: A New ENGland Folk Tale - T..[LimeTorrents][0m
05-05 02:59:31 INFO [0m[edia._base.providers.base] Found: score(65) on TorrentPotato, 10.0.0.12: Czarownica: Bajka ludowa z Nowej Anglii - The Witch: A New ENGland Folk Tale - T..[LimeTorrents][0m
05-05 02:59:32 INFO [0m[edia._base.providers.base] Found: score(2572) on TorrentPotato, 10.0.0.12: The.Witch.2016.720p.BluRay.x264-NeZu[LimeTorrents][0m
05-05 02:59:32 INFO [0m[edia._base.providers.base] Found: score(1144) on TorrentPotato, 10.0.0.12: Mark of the Witch 2016 HDRip XviD AC3-EVO[LimeTorrents][0m
05-05 02:59:32 INFO [0m[edia._base.providers.base] Found: score(432) on TorrentPotato, 10.0.0.12: Mark of the Witch 2016 HDRip XviD AC3-EVO[LimeTorrents][0m
05-05 02:59:32 INFO [0m[edia._base.providers.base] Found: score(853) on TorrentPotato, 10.0.0.12: The Witch 2016 HDRip DS X264-VoMiT[LimeTorrents][0m
05-05 02:59:32 INFO [0m[edia._base.providers.base] Found: score(504) on TorrentPotato, 10.0.0.12: Mark of the Witch 2016 HDRip XviD AC3-EVO[LimeTorrents][0m
05-05 02:59:32 INFO [0m[edia._base.providers.base] Found: score(586) on TorrentPotato, 10.0.0.12: The Witch 2016 HDRip x264 AAC-RARBG[LimeTorrents][0m
05-05 02:59:32 INFO [0m[edia._base.providers.base] Found: score(663) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:33 INFO [0m[edia._base.providers.base] Found: score(678) on TorrentPotato, 10.0.0.12: The Witch 2016 1080p HDRip x264 AC3 6CH-CPG[LimeTorrents][0m
05-05 02:59:33 INFO [0m[edia._base.providers.base] Found: score(659) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:33 INFO [0m[edia._base.providers.base] Found: score(432) on TorrentPotato, 10.0.0.12: Mark of the Witch 2016 HDRip XviD AC3-EVO[LimeTorrents][0m
05-05 02:59:33 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:33 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:33 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:33 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:33 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:34 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:34 INFO [0m[edia._base.providers.base] Found: score(563) on TorrentPotato, 10.0.0.12: The Witch 2016 HDRip 720p x264 -DIMENSION[LimeTorrents][0m
05-05 02:59:34 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:34 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:34 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:34 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:34 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:34 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:35 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:35 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:35 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:35 INFO [0m[edia._base.providers.base] Found: score(573) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BluRay x264- RARBG[LimeTorrents][0m
05-05 02:59:35 INFO [0m[edia._base.providers.base] Found: score(563) on TorrentPotato, 10.0.0.12: The Witch 2016 HDRip 720p x264 -DIMENSION[LimeTorrents][0m
05-05 02:59:35 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:36 INFO [0m[edia._base.providers.base] Found: score(643) on TorrentPotato, 10.0.0.12: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:36 INFO [0m[edia._base.providers.base] Found: score(378) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.BluRay.H264.AAC[MagnetDL][0m
05-05 02:59:36 INFO [0m[edia._base.providers.base] Found: score(378) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BluRay.H264.AAC[MagnetDL][0m
05-05 02:59:36 INFO [0m[edia._base.providers.base] Found: score(354) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.MP3-XVID[MagnetDL][0m
05-05 02:59:36 INFO [0m[edia._base.providers.base] Found: score(378) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BluRay.x264-x0r [PRiME][MagnetDL][0m
05-05 02:59:36 INFO [0m[edia._base.providers.base] Found: score(403) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BRRip.x264.AAC-ETRG[MagnetDL][0m
05-05 02:59:37 INFO [0m[edia._base.providers.base] Found: score(315) on TorrentPotato, 10.0.0.12: 18 + The Love Witch 2016 UNCENSORED Movies 720p BluRay X264 New[MagnetDL][0m
05-05 02:59:37 INFO [0m[edia._base.providers.base] Found: score(393) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD [EtHD][MagnetDL][0m
05-05 02:59:37 INFO [0m[edia._base.providers.base] Found: score(368) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD [PRiME][MagnetDL][0m
05-05 02:59:37 INFO [0m[edia._base.providers.base] Found: score(393) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD [PRiME][MagnetDL][0m
05-05 02:59:37 INFO [0m[edia._base.providers.base] Found: score(374) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD[MagnetDL][0m
05-05 02:59:37 INFO [0m[edia._base.providers.base] Found: score(368) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD[MagnetDL][0m
05-05 02:59:37 INFO [0m[edia._base.providers.base] Found: score(363) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.BRRip.x264.AAC-ETRG[MagnetDL][0m
05-05 02:59:38 INFO [0m[edia._base.providers.base] Found: score(378) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.720p.BluRay.x264-EiDER [EtHD][MagnetDL][0m
05-05 02:59:38 INFO [0m[edia._base.providers.base] Found: score(353) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.LiMiTED.BDRip.x264-EiDER[MagnetDL][0m
05-05 02:59:38 INFO [0m[edia._base.providers.base] Found: score(354) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[MagnetDL][0m
05-05 02:59:38 INFO [0m[edia._base.providers.base] Found: score(360) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[MagnetDL][0m
05-05 02:59:38 INFO [0m[edia._base.providers.base] Found: score(360) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[MagnetDL][0m
05-05 02:59:38 INFO [0m[edia._base.providers.base] Found: score(360) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.BRRip.XviD.AC3-EVO [PRiME][MagnetDL][0m
05-05 02:59:38 INFO [0m[edia._base.providers.base] Found: score(368) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.720p.WEB-DL.DD5.1.x264-BDP [PRiME][MagnetDL][0m
05-05 02:59:38 INFO [0m[edia._base.providers.base] Found: score(358) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.DD2.0.x264-BDP [PRiME][MagnetDL][0m
05-05 02:59:39 INFO [0m[edia._base.providers.base] Found: score(358) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.DD2.0.x264-BDP[MagnetDL][0m
05-05 02:59:39 INFO [0m[edia._base.providers.base] Found: score(367) on TorrentPotato, 10.0.0.12: The Love Witch (2016) 720p WEB-DL 950MB: MkvCage[MagnetDL][0m
05-05 02:59:39 INFO [0m[edia._base.providers.base] Found: score(367) on TorrentPotato, 10.0.0.12: The Love Witch (2016) 720p WEB-DL: 1.2GB: ShAaNiG[MagnetDL][0m
05-05 02:59:39 INFO [0m[edia._base.providers.base] Found: score(304) on TorrentPotato, 10.0.0.12: 18 + The Love Witch 2016 UNCENSORED Movies 720p HDRip XviD ESubs[MagnetDL][0m
05-05 02:59:39 INFO [0m[edia._base.providers.base] Found: score(353) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-iFT[MagnetDL][0m
05-05 02:59:39 INFO [0m[edia._base.providers.base] Found: score(359) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[MagnetDL][0m
05-05 02:59:39 INFO [0m[edia._base.providers.base] Found: score(386) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[MagnetDL][0m
05-05 02:59:39 INFO [0m[edia._base.providers.base] Found: score(353) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO [PRiME][MagnetDL][0m
05-05 02:59:39 INFO [0m[edia._base.providers.base] Found: score(353) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-iFT [PRiME][MagnetDL][0m
05-05 02:59:40 INFO [0m[edia._base.providers.base] Found: score(353) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[MagnetDL][0m
05-05 02:59:40 INFO [0m[edia._base.providers.base] Found: score(368) on TorrentPotato, 10.0.0.12: The.Love.Witch.2016.1080p.WEB-DL.DD5.1.H264-FGT [EtHD][MagnetDL][0m
05-05 02:59:40 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/t/the-witch-2016/se/desc/1/, data: [][0m
05-05 02:59:40 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22The+Witch+2016%22/0/7/201,207, data: [][0m
05-05 02:59:42 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/torrent/13549839/Mark.of.the.Witch.2016.HDRip.XviD.AC3-EVO, data: [][0m
05-05 02:59:43 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/bWFyay5vZi50aGUud2l0Y2guMjAxNi5oZHJpcC54dmlkLmFjMy1ldm8=/, data: [][0m
05-05 02:59:43 INFO [0m[edia._base.providers.base] Found: score(209) on ThePirateBay: Mark.of.the.Witch.2016.HDRip.XviD.AC3-EVO[0m
05-05 02:59:43 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/torrent/13575458/18__Mark_Of_The_Witch_2016_Movies_HDRip_XviD_New__Sample____rDX_, data: [][0m
05-05 02:59:45 INFO [0m[edia._base.providers.base] Found: score(114) on ThePirateBay: 18+ Mark Of The Witch 2016 Movies HDRip XviD New +Sample ~ ☻rDX☻[0m
05-05 02:59:45 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/torrent/14360949/The_Witch_2016_MP4, data: [][0m
05-05 02:59:46 INFO [0m[edia._base.providers.base] Found: score(196) on ThePirateBay: The Witch 2016 MP4[0m
05-05 02:59:46 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/torrent/13548920/Mark_of_the_Witch_2016_HDRip_XviD_AC3-EVO, data: [][0m
05-05 02:59:47 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/bWFyayBvZiB0aGUgd2l0Y2ggMjAxNiBoZHJpcCB4dmlkIGFjMy1ldm8=/, data: [][0m
05-05 02:59:47 INFO [0m[edia._base.providers.base] Found: score(166) on ThePirateBay: Mark of the Witch 2016 HDRip XviD AC3-EVO[0m
05-05 02:59:48 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/torrent/13551842/Mark_of_the_Witch_2016_HDRip_XviD_AC3-EVO_, data: [][0m
05-05 02:59:49 INFO [0m[edia._base.providers.base] Found: score(156) on ThePirateBay: Mark of the Witch 2016 HDRip XviD AC3-EVO [0m
05-05 02:59:49 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/torrent/14520409/The.Witch.2016.720p.BluRay.x264-NeZu, data: [][0m
05-05 02:59:51 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTYuNzIwcC5ibHVyYXkueDI2NC1uZXp1/, data: [][0m
05-05 02:59:51 INFO [0m[edia._base.providers.base] Found: score(321) on ThePirateBay: The.Witch.2016.720p.BluRay.x264-NeZu[0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 HDRip x264 AC3 6CH-CPG[1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[PRiME][1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.720p.BluRay.H264.AAC[1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.720p.WEB-DL.XviD.AC3-FGT[1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.720p.WEB-DL.x264 - WeTv[1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.1080p.WEB-DL.x264 - WeTv[1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.LiMiTED.BDRip.x264-EiDER[1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[PRiME][1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: Mark of the Witch 2016 HDRip XviD AC3-EVO[1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: 18+ The Love Witch 2016 UNCENSORED Movies 720p HDRip XviD ESubs AAC New +Sample ☻rDX☻[1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: 18+ Mark Of The Witch 2016 English Movies HDRip XviD AAC New Source with Sample ~ ☻rDX☻[1337x][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.PROPER.720p.BluRay.x264-PSYCHD[PRiME][Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.720p.BluRay.H264.AAC[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.720p.BRRip.x264.AAC-ETRG[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.720p.WEB-DL.XviD.AC3-FGT[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.720p.WEB-DL.x264 - WeTv[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.LiMiTED.BDRip.x264-EiDER[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-iFT[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-iFT[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[PRiME][Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.DD2.0.x264-BDP[PRiME][Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: Mark of the Witch 2016 HDRip XviD AC3-EVO[Isohunt2][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 (1080p x265 10bit Joy)[KickAssTorrent][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BRRip XviD-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 720p BluRay x264- RARBG[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 HDRip 720p x264 -DIMENSION[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The Witch 2016 HDRip 720p x264 -DIMENSION[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: Mark of the Witch 2016 HDRip XviD AC3 EVO[KickAssTorrent][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: Mark of the Witch 2016 HDRip XviD AC3-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: Mark of the Witch 2016 HDRip XviD AC3-EVO[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD [PRiME][MagnetDL][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.PROPER.BDRip.x264-PSYCHD[MagnetDL][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.720p.BRRip.x264.AAC-ETRG[MagnetDL][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.BRRip.XviD.AC3-EVO[MagnetDL][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.LiMiTED.BDRip.x264-EiDER[MagnetDL][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-iFT[MagnetDL][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-EVO [PRiME][MagnetDL][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-iFT [PRiME][MagnetDL][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Love.Witch.2016.HDRip.XviD.AC3-EVO[MagnetDL][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: The.Witch.2016.720p.BluRay.x264-NeZu[0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: Mark of the Witch 2016 HDRip XviD AC3-EVO[0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: Mark of the Witch 2016 HDRip XviD AC3-EVO [0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: Czarownica: Bajka ludowa z Nowej Anglii - The Witch: A New ENGland Folk Tale - T..[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: Czarownica: Bajka ludowa z Nowej Anglii - The Witch: A New ENGland Folk Tale - T..[LimeTorrents][0m
05-05 02:59:53 INFO [0m[tato.core.plugins.release] Tried to download, but none of the "torrent" downloaders are enabled or gave an error[0m
05-05 02:59:53 INFO [0m[core.media.movie.searcher] Search for The Witch in BR-Disk ignoring ETA[0m
05-05 02:59:53 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt4263482&adv_nfo=on&xminsize=20000&adv_age=1500&max=400&xmaxsize=60000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 02:59:54 INFO [0m[edia._base.providers.base] Found: score(1516) on Newznab, api.nzbgeek.info: The.VVitch.A.New-England.Folktale.2015.Blu-ray.CEE.1080p.AVC.DTS.5.1-HDCLUB THE WITCH HDCLUB[0m
05-05 02:59:54 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLmEubmV3LWVuZ2xhbmQuZm9sa3RhbGUuMjAxNS5ibHUtcmF5LmNlZS4xMDgwcC5hdmMuZHRzLjUuMS1oZGNsdWI=/, data: [][0m
05-05 02:59:54 INFO [0m[edia._base.providers.base] Found: score(950) on Newznab, api.nzbgeek.info: The.Witch.A.New-England.Folktale.2015.Blu-ray.CEE.1080p.AVC.DTS.5.1-HDCLUB[0m
05-05 02:59:55 INFO [0m[edia._base.providers.base] Found: score(1904) on Newznab, api.nzbgeek.info: The.Witch.2015.BluRay.Remux.1080p.DTS-HD.MA.5.1-HiFi.cp(tt4263482)[0m
05-05 02:59:55 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUuYmx1cmF5LnJlbXV4LjEwODBwLmR0cy1oZC5tYS41LjEtaGlmaQ==/, data: [][0m
05-05 02:59:55 INFO [0m[edia._base.providers.base] Found: score(1904) on Newznab, api.nzbgeek.info: The.Witch.2015.BluRay.Remux.1080p.DTS-HD.MA.5.1-HiFi[0m
05-05 02:59:55 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 02:59:55 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:59:56 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 02:59:56 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 02:59:56 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:59:56 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 02:59:56 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:59:56 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:59:57 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/z/zombieland-2-2019/se/desc/1/, data: [][0m
05-05 02:59:57 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Zombieland+2+2019%22/0/7/201, data: [][0m
05-05 02:59:58 INFO [0m[edia._base.providers.base] Found: score(5268) on TorrentPotato, 10.0.0.12: The Love Witch 2016 1080p BluRay REMUX AVC DTS-HD MA 5 1-FGT[KATcrs][0m
05-05 02:59:58 INFO [0m[core.media.movie.searcher] Search for Zombieland 2 in DVD-R ignoring ETA[0m
05-05 02:59:58 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 02:59:58 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:59:58 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:59:58 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:59:58 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 02:59:58 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 02:59:58 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 02:59:58 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 02:59:59 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/t/the-witch-2016/se/desc/1/, data: [][0m
05-05 02:59:59 INFO [0m[tato.core.plugins.release] Tried to download, but none of the "torrent" downloaders are enabled or gave an error[0m
05-05 02:59:59 INFO [0m[core.media.movie.searcher] Search for The Witch in DVD-Rip ignoring ETA[0m
05-05 02:59:59 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt4263482&adv_nfo=on&xminsize=600&adv_age=1500&max=400&xmaxsize=2400&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:00:01 INFO [0m[core.media._base.searcher] Wrong: 02 18 7ls5dowka3 nfo 650 03 mb www nfo underground xxx have fun und besucht uns unter www nfo underground xxx yenc 1 1, probably pr0n[0m
05-05 03:00:01 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:00:01 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:00:01 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:00:01 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:00:01 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 03:00:02 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 03:00:02 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:00:02 INFO [0m[edia._base.providers.base] Found: score(807) on TorrentPotato, 10.0.0.12: The Witch VOSTFR DVDRIP 2016[Isohunt2][0m
05-05 03:00:03 INFO [0m[edia._base.providers.base] Found: score(407) on TorrentPotato, 10.0.0.12: The Witch VOSTFR DVDRIP 2016[MagnetDL][0m
05-05 03:00:04 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/t/the-witch-2016/se/desc/1/, data: [][0m
05-05 03:00:04 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22The+Witch+2016%22/0/7/201, data: [][0m
05-05 03:00:06 INFO [0m[tato.core.plugins.release] Tried to download, but none of the "torrent" downloaders are enabled or gave an error[0m
05-05 03:00:06 INFO [0m[core.media.movie.searcher] Search for The Witch in DVD-R ignoring ETA[0m
05-05 03:00:07 INFO [0m[edia._base.providers.base] Found: score(26) on BinSearch: 01 51 7th8naxck45kb3ccgzu6 par2 4 41 gb yenc 1 1[0m
05-05 03:00:07 INFO [0m[edia._base.providers.base] Found: score(181) on BinSearch: 01 47 88853 20160522 006 par2 3 88 gb yenc 1 1[0m
05-05 03:00:07 INFO [0m[edia._base.providers.base] Found: score(1879) on Newznab, api.nzbgeek.info: The.Witch.PAL.DVDR-iGNiTiON[0m
05-05 03:00:07 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/dGhlLndpdGNoLjIwMTUubnRzYy5kdmRyLXAycA==/, data: [][0m
05-05 03:00:08 INFO [0m[edia._base.providers.base] Found: score(2019) on Newznab, api.nzbgeek.info: The.Witch.2015.NTSC.DVDR-P2P[0m
05-05 03:00:08 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:00:08 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:00:08 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:00:08 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:00:09 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 03:00:09 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt4263482&search=The+Witch+2016&user=&passkey=xxx[0m
05-05 03:00:09 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:00:10 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/t/the-witch-2016/se/desc/1/, data: [][0m
05-05 03:00:10 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22The+Witch+2016%22/0/7/202, data: [][0m
05-05 03:00:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.nzbgeek.info/api?t=get&id=95165a6c53b73d0777181480fc6ee82d&apikey=xxx[0m
05-05 03:00:14 INFO [0m[.core.downloaders.sabnzbd] Sending "The.Witch.2015.NTSC.DVDR-P2P" to SABnzbd.[0m
05-05 03:00:14 INFO [0m[hpotato.core.plugins.base] Opening url: post http://localhost:8080/api?nzbname=The.Witch.2015.NTSC.DVDR-P2P.cp%28tt4263482%29&apikey=xxx&cat=couchpotato&priority=0&mode=addfile&output=json, data: [][0m
05-05 03:00:18 INFO [0m[.core.downloaders.sabnzbd] NZB sent to SAB successfully.[0m
05-05 03:00:18 INFO [0m[tato.core.plugins.release] Snatched "The.Witch.2015.NTSC.DVDR-P2P": The Witch (2016) in dvdr from Newznab, api.nzbgeek.info[0m
05-05 03:00:28 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 03:00:28 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/z/zombieland-2-2019/se/desc/1/, data: [][0m
05-05 03:00:28 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Zombieland+2+2019%22/0/7/202, data: [][0m
05-05 03:00:29 INFO [0m[core.media.movie.searcher] Search for Zombieland 2 in Screener ignoring ETA[0m
05-05 03:00:29 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt1560220&adv_nfo=on&xminsize=600&adv_age=1500&max=400&xmaxsize=1600&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:00:30 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:00:30 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 03:00:30 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:00:30 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 03:00:30 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:00:30 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 03:00:31 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:00:31 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:01:00 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt1560220&search=Zombieland+2+2019&user=&passkey=xxx[0m
05-05 03:01:00 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/z/zombieland-2-2019/se/desc/1/, data: [][0m
05-05 03:01:02 INFO [0m[core.media.movie.searcher] Search for John Wick: Chapter 3 – Parabellum in 720p ignoring ETA[0m
05-05 03:01:02 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.nzbgeek.info/api?imdbid=6146586&apikey=xxx&t=movie&extended=1&cat=2000, data: [][0m
05-05 03:01:02 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt6146586&adv_nfo=on&xminsize=3000&adv_age=1500&max=400&xmaxsize=10000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:01:04 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:01:04 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/gimmepeers/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:01:04 INFO [0m[hpotato.core.plugins.base] Opening url: get https://hd4free.xyz/searchapi.php?apikey=&username=&imdbid=tt6146586&internal=False, data: [][0m
05-05 03:01:04 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:01:04 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:01:05 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:01:08 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/1337x/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:01:11 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:01:11 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:01:12 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/anidex/results/torznab/api?apikey=xxx&t=search&cat=&q=?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:01:30 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:01:31 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/isohunt2/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:01:35 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:05 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:05 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:11 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/limetorrents/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:12 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/magnetdl/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:14 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/nyaasi/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:15 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:02:15 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019%22/0/7/207, data: [][0m
05-05 03:02:15 INFO [0m[hpotato.core.plugins.base] Opening url: get https://yts.am/api/v2/list_movies.json?query_term=tt6146586&limit=10&page=1, data: [][0m
05-05 03:02:15 INFO [0m[hpotato.core.plugins.base] Opening url: get http://katcr.to/span/m-i6146586/, data: [][0m
05-05 03:02:16 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:02:17 ERROR [31m[hpotato.core.plugins.base] Failed opening url in MagnetDL: http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/ Traceback (most recent call last):
HTTPError: 404 Client Error: Not Found for url: https://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/
[0m
05-05 03:02:17 INFO [0m[core.media.movie.searcher] Search for John Wick: Chapter 3 – Parabellum in 1080p ignoring ETA[0m
05-05 03:02:17 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt6146586&adv_nfo=on&xminsize=4000&adv_age=1500&max=400&xmaxsize=20000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:02:18 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:02:18 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:18 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:02:18 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:18 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:02:18 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:19 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:02:19 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:02:48 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:48 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:02:49 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:02:49 ERROR [31m[hpotato.core.plugins.base] Failed opening url in MagnetDL: http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/ Traceback (most recent call last):
HTTPError: 404 Client Error: Not Found for url: https://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/
[0m
05-05 03:02:49 INFO [0m[core.media.movie.searcher] Search for John Wick: Chapter 3 – Parabellum in BR-Rip ignoring ETA[0m
05-05 03:02:49 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt6146586&adv_nfo=on&xminsize=700&adv_age=1500&max=400&xmaxsize=7000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:02:50 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:02:50 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:50 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:02:50 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:50 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:02:50 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:02:50 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:02:50 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:03:20 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:03:20 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:03:20 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019%22/0/7/201,207, data: [][0m
05-05 03:03:23 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:03:23 ERROR [31m[hpotato.core.plugins.base] Failed opening url in MagnetDL: http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/ Traceback (most recent call last):
HTTPError: 404 Client Error: Not Found for url: https://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/
[0m
05-05 03:03:23 INFO [0m[core.media.movie.searcher] Search for John Wick: Chapter 3 – Parabellum in BR-Disk ignoring ETA[0m
05-05 03:03:23 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt6146586&adv_nfo=on&xminsize=20000&adv_age=1500&max=400&xmaxsize=60000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:03:24 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:03:24 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:03:24 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:03:24 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:03:24 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:03:24 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:03:24 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:03:24 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:03:46 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:03:48 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:03:48 ERROR [31m[hpotato.core.plugins.base] Failed opening url in MagnetDL: http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/ Traceback (most recent call last):
HTTPError: 404 Client Error: Not Found for url: https://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/
[0m
05-05 03:03:48 INFO [0m[core.media.movie.searcher] Search for John Wick: Chapter 3 – Parabellum in DVD-Rip ignoring ETA[0m
05-05 03:03:48 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt6146586&adv_nfo=on&xminsize=600&adv_age=1500&max=400&xmaxsize=2400&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:03:49 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:03:49 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:03:49 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:03:49 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:03:49 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:03:49 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:03:49 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:03:49 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:03:49 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019%22/0/7/201, data: [][0m
05-05 03:03:50 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:03:50 ERROR [31m[hpotato.core.plugins.base] Failed opening url in MagnetDL: http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/ Traceback (most recent call last):
HTTPError: 404 Client Error: Not Found for url: https://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/
[0m
05-05 03:03:50 INFO [0m[core.media.movie.searcher] Search for John Wick: Chapter 3 – Parabellum in DVD-R ignoring ETA[0m
05-05 03:03:50 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:03:50 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:03:50 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:03:50 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:03:50 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:03:51 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:03:51 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:03:51 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019%22/0/7/202, data: [][0m
05-05 03:03:51 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:03:52 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:03:52 ERROR [31m[hpotato.core.plugins.base] Failed opening url in MagnetDL: http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/ Traceback (most recent call last):
HTTPError: 404 Client Error: Not Found for url: https://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/
[0m
05-05 03:03:52 INFO [0m[core.media.movie.searcher] Search for John Wick: Chapter 3 – Parabellum in Screener ignoring ETA[0m
05-05 03:03:52 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt6146586&adv_nfo=on&xminsize=600&adv_age=1500&max=400&xmaxsize=1600&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:03:53 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:03:53 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:03:53 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:03:53 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt6146586&search=John+Wick%3A+Chapter+3+%E2%80%93+Parabellum+2019&user=&passkey=xxx[0m
05-05 03:03:53 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:03:53 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:03:53 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:03:53 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:03:55 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/, data: [][0m
05-05 03:03:55 ERROR [31m[hpotato.core.plugins.base] Failed opening url in MagnetDL: http://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/ Traceback (most recent call last):
HTTPError: 404 Client Error: Not Found for url: https://www.magnetdl.com/j/john-wick-chapter-3-%E2%80%93-parabellum-2019/se/desc/1/
[0m
05-05 03:03:55 ERROR [31m[to.core.media._base.media] Media with id "6b5c9f4045f742a393d0b15e58bd8332" not found[0m
05-05 03:03:55 ERROR [31m[to.core.media._base.media] Media with id "c54bb6954cee43ab93e1f3cd5fb6980d" not found[0m
05-05 03:03:55 INFO [0m[core.media.movie.searcher] Search for Hellboy in Screener[0m
05-05 03:03:55 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.nzbgeek.info/api?imdbid=2274648&apikey=xxx&t=movie&extended=1&cat=2000, data: [][0m
05-05 03:03:57 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt2274648&adv_nfo=on&xminsize=600&adv_age=1500&max=400&xmaxsize=1600&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:03:58 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:03:58 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/gimmepeers/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:03:58 INFO [0m[hpotato.core.plugins.base] Opening url: get https://hd4free.xyz/searchapi.php?apikey=&username=&imdbid=tt2274648&internal=False, data: [][0m
05-05 03:03:58 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:03:58 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:03:59 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:04:03 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/1337x/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:04:08 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:04:08 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:04:08 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/anidex/results/torznab/api?apikey=xxx&t=search&cat=&q=?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:04:29 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:04:30 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/isohunt2/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:04:34 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:05:04 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:05:04 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:05:06 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/limetorrents/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:05:08 INFO [0m[edia._base.providers.base] Found: score(872) on TorrentPotato, 10.0.0.12: Hellboy (2019)[720p - HQ DVDScr - HQ Line Audios - [Tamil + Eng] - x264 - 900MB][LimeTorrents][0m
05-05 03:05:08 INFO [0m[edia._base.providers.base] Found: score(744) on TorrentPotato, 10.0.0.12: Hellboy (2019)[720p - HQ DVDScr - HQ Line Audios - [Tamil + Telugu + Hindi + Eng..[LimeTorrents][0m
05-05 03:05:08 INFO [0m[edia._base.providers.base] Found: score(647) on TorrentPotato, 10.0.0.12: Hellboy (2019)[HQ DVDScr - HQ Line Audios - [Tamil + Telugu] - XviD - MP3 - 700M..[LimeTorrents][0m
05-05 03:05:10 INFO [0m[edia._base.providers.base] Found: score(458) on TorrentPotato, 10.0.0.12: Hellboy (2019)[720p - HQ DVDScr - HQ Line Audios - [Tamil + Telugu + Eng] - x26..[LimeTorrents][0m
05-05 03:05:10 INFO [0m[edia._base.providers.base] Found: score(418) on TorrentPotato, 10.0.0.12: Hellboy (2019)[720p - HQ DVDScr - HQ Line Audios - [Hindi + Eng] - x264 - 800MB][LimeTorrents][0m
05-05 03:05:11 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/magnetdl/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:05:13 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/nyaasi/results/potato/?imdbid=tt2274648&search=Hellboy+2019&user=&passkey=xxx[0m
05-05 03:05:14 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Hellboy+2019%22/0/7/201, data: [][0m
05-05 03:05:14 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/h/hellboy-2019/se/desc/1/, data: [][0m
05-05 03:05:14 INFO [0m[hpotato.core.plugins.base] Opening url: get https://yts.am/api/v2/list_movies.json?query_term=tt2274648&limit=10&page=1, data: [][0m
05-05 03:05:14 INFO [0m[hpotato.core.plugins.base] Opening url: get http://katcr.to/span/m-i2274648/, data: [][0m
05-05 03:05:16 INFO [0m[tato.core.plugins.release] Tried to download, but none of the "torrent" downloaders are enabled or gave an error[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "3be5382add9b4da18019a1937ed75f7c" not found[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "bda41bbd1e5248b1bd61dd9a31d57b3c" not found[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "57602ce57b93446b9b5ef7bb8b4281fe" not found[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "85468c46abfc4e8e9afc6ff3a0a21b60" not found[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "3d9485b9a90b426fa75988dd5c4c03e2" not found[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "143f488d5d964f67a95e47d1ba6efcc0" not found[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "ee343c757c01452d85163f8965774ec5" not found[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "6917988ff03647408886f00b318e1ec6" not found[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "022e76a2462647119a834a8db8a30d4e" not found[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "5da4576eecd343cb872a44030cd11513" not found[0m
05-05 03:05:20 ERROR [31m[to.core.media._base.media] Media with id "232cf7cc94aa4e7490a547f676306bc7" not found[0m
05-05 03:05:20 INFO [0m[core.media.movie.searcher] Search for Midsommar in Screener[0m
05-05 03:05:20 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt8772262&adv_nfo=on&xminsize=600&adv_age=1500&max=400&xmaxsize=1600&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:05:20 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.nzbgeek.info/api?imdbid=8772262&apikey=xxx&t=movie&extended=1&cat=2000, data: [][0m
05-05 03:05:22 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:05:22 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/gimmepeers/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:05:22 INFO [0m[hpotato.core.plugins.base] Opening url: get https://hd4free.xyz/searchapi.php?apikey=&username=&imdbid=tt8772262&internal=False, data: [][0m
05-05 03:05:23 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:05:23 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:05:23 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:05:28 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/1337x/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:05:31 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:05:31 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:05:32 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/anidex/results/torznab/api?apikey=xxx&t=search&cat=&q=?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:05:52 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:05:53 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/isohunt2/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:05:56 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:06:27 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:06:27 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:06:32 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/limetorrents/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:06:33 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/magnetdl/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:06:35 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/nyaasi/results/potato/?imdbid=tt8772262&search=Midsommar+2019&user=&passkey=xxx[0m
05-05 03:06:36 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/m/midsommar-2019/se/desc/1/, data: [][0m
05-05 03:06:36 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Midsommar+2019%22/0/7/201, data: [][0m
05-05 03:06:36 INFO [0m[hpotato.core.plugins.base] Opening url: get https://yts.am/api/v2/list_movies.json?query_term=tt8772262&limit=10&page=1, data: [][0m
05-05 03:06:36 INFO [0m[hpotato.core.plugins.base] Opening url: get http://katcr.to/span/m-i8772262/, data: [][0m
05-05 03:06:38 INFO [0m[core.media.movie.searcher] Search for Ad Astra in Screener[0m
05-05 03:06:38 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.nzbgeek.info/api?imdbid=2935510&apikey=xxx&t=movie&extended=1&cat=2000, data: [][0m
05-05 03:06:38 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt2935510&adv_nfo=on&xminsize=600&adv_age=1500&max=400&xmaxsize=1600&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:06:40 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:06:40 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/gimmepeers/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:06:40 INFO [0m[hpotato.core.plugins.base] Opening url: get https://hd4free.xyz/searchapi.php?apikey=&username=&imdbid=tt2935510&internal=False, data: [][0m
05-05 03:06:41 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:06:41 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:06:41 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:06:45 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/1337x/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:06:48 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:06:48 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:06:49 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/anidex/results/torznab/api?apikey=xxx&t=search&cat=&q=?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:07:12 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:07:13 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/isohunt2/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:07:15 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:07:38 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:07:43 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/limetorrents/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:07:45 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/magnetdl/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:07:46 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/nyaasi/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
05-05 03:07:47 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/a/ad-astra-2019/se/desc/1/, data: [][0m
05-05 03:07:47 INFO [0m[hpotato.core.plugins.base] Opening url: get https://torrentapi.org/pubapi_v2.php?get_token=get_token&app_id=couchpotato, data: [][0m
05-05 03:07:47 INFO [0m[hpotato.core.plugins.base] Opening url: get https://yts.am/api/v2/list_movies.json?query_term=tt2935510&limit=10&page=1, data: [][0m
05-05 03:07:47 INFO [0m[hpotato.core.plugins.base] Opening url: get http://katcr.to/span/m-i2935510/, data: [][0m
05-05 03:07:47 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Ad+Astra+2019%22/0/7/201, data: [][0m
05-05 03:07:50 INFO [0m[hpotato.core.plugins.base] Opening url: get https://torrentapi.org/pubapi_v2.php?token=lt1jzcqb3x&mode=search&search_imdb=tt2935510&min_seeders=10&min_leechers=0&ranked=1&category=movies&format=json_extended&app_id=couchpotato, data: [][0m
05-05 03:07:50 ERROR [31mFailed encoding stuff to log "Cant find imdb in database. Are you sure this imdb exists?": not all arguments converted during string formatting[0m
05-05 03:07:50 ERROR [31m[e.providers.torrent.rarbg] Cant find imdb in database. Are you sure this imdb exists?[0m
05-05 03:07:50 INFO [0m[core.media.movie.searcher] Search for Star Wars: Episode IX in 720p ignoring ETA[0m
05-05 03:07:50 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt2527338&adv_nfo=on&xminsize=3000&adv_age=1500&max=400&xmaxsize=10000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:07:50 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.nzbgeek.info/api?imdbid=2527338&apikey=xxx&t=movie&extended=1&cat=2000, data: [][0m
05-05 03:07:52 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:07:52 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/gimmepeers/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:07:52 INFO [0m[hpotato.core.plugins.base] Opening url: get https://hd4free.xyz/searchapi.php?apikey=&username=&imdbid=tt2527338&internal=False, data: [][0m
05-05 03:07:54 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:07:54 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:07:55 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:07:58 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/1337x/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:08:02 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:08:02 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:08:03 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/anidex/results/torznab/api?apikey=xxx&t=search&cat=&q=?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:08:25 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:08:26 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/isohunt2/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:08:28 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:08:58 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:09:20 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:09:25 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/limetorrents/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:09:27 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/magnetdl/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:09:29 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/nyaasi/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:09:31 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:09:31 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Star+Wars%3A+Episode+IX+2019%22/0/7/207, data: [][0m
05-05 03:09:31 INFO [0m[hpotato.core.plugins.base] Opening url: get http://katcr.to/span/m-i2527338/, data: [][0m
05-05 03:09:31 INFO [0m[hpotato.core.plugins.base] Opening url: get https://yts.am/api/v2/list_movies.json?query_term=tt2527338&limit=10&page=1, data: [][0m
05-05 03:09:31 INFO [0m[hpotato.core.plugins.base] Opening url: get https://torrentapi.org/pubapi_v2.php?token=lt1jzcqb3x&mode=search&search_imdb=tt2527338&min_seeders=10&min_leechers=0&ranked=1&category=movies&format=json_extended&app_id=couchpotato, data: [][0m
05-05 03:09:32 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:09:32 INFO [0m[core.media.movie.searcher] Search for Star Wars: Episode IX in 1080p ignoring ETA[0m
05-05 03:09:32 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt2527338&adv_nfo=on&xminsize=4000&adv_age=1500&max=400&xmaxsize=20000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:09:33 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:09:33 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:09:33 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:09:33 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:09:33 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:09:33 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:09:34 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:09:34 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:10:03 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:10:03 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:10:05 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:10:05 INFO [0m[core.media.movie.searcher] Search for Star Wars: Episode IX in BR-Rip ignoring ETA[0m
05-05 03:10:05 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt2527338&adv_nfo=on&xminsize=700&adv_age=1500&max=400&xmaxsize=7000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:10:05 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:10:05 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:10:05 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:10:06 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:10:06 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:10:06 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:10:06 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:10:06 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:10:36 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:10:36 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:10:36 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Star+Wars%3A+Episode+IX+2019%22/0/7/201,207, data: [][0m
05-05 03:10:38 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:10:38 INFO [0m[core.media.movie.searcher] Search for Star Wars: Episode IX in BR-Disk ignoring ETA[0m
05-05 03:10:38 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt2527338&adv_nfo=on&xminsize=20000&adv_age=1500&max=400&xmaxsize=60000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:10:39 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:10:39 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:10:39 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:10:39 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:10:39 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:10:39 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:10:39 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:10:39 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:11:09 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:11:09 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:11:10 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:11:10 INFO [0m[core.media.movie.searcher] Search for Star Wars: Episode IX in DVD-Rip ignoring ETA[0m
05-05 03:11:10 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.nzbclub.com/nzbrss.aspx?q=Star+Wars%3A+Episode+IX+2019&rpp=200&sp=1&ns=1&ig=1&st=5, data: [][0m
05-05 03:11:10 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt2527338&adv_nfo=on&xminsize=600&adv_age=1500&max=400&xmaxsize=2400&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:11:41 ERROR [31m[hpotato.core.plugins.base] Failed opening url in NZBClub: https://www.nzbclub.com/nzbrss.aspx?q=Star+Wars%3A+Episode+IX+2019&rpp=200&sp=1&ns=1&ig=1&st=5 Traceback (most recent call last):
ConnectTimeout: HTTPSConnectionPool(host='www.nzbclub.com', port=443): Max retries exceeded with url: /nzbrss.aspx?q=Star+Wars%3A+Episode+IX+2019&rpp=200&sp=1&ns=1&ig=1&st=5 (Caused by ConnectTimeoutError(<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0xb45a4750>, 'Connection to www.nzbclub.com timed out. (connect timeout=30)'))
[0m
05-05 03:11:41 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:11:41 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:11:41 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:11:41 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:11:41 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:11:41 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:11:42 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:11:42 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:12:11 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:12:11 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:12:11 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Star+Wars%3A+Episode+IX+2019%22/0/7/201, data: [][0m
05-05 03:12:12 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:12:13 INFO [0m[core.media.movie.searcher] Search for Star Wars: Episode IX in DVD-R ignoring ETA[0m
05-05 03:12:13 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.nzbclub.com/nzbrss.aspx?q=Star+Wars%3A+Episode+IX+2019&rpp=200&sp=1&ns=1&ig=1&st=5, data: [][0m
05-05 03:12:43 ERROR [31m[hpotato.core.plugins.base] Failed opening url in NZBClub: https://www.nzbclub.com/nzbrss.aspx?q=Star+Wars%3A+Episode+IX+2019&rpp=200&sp=1&ns=1&ig=1&st=5 Traceback (most recent call last):
ConnectTimeout: HTTPSConnectionPool(host='www.nzbclub.com', port=443): Max retries exceeded with url: /nzbrss.aspx?q=Star+Wars%3A+Episode+IX+2019&rpp=200&sp=1&ns=1&ig=1&st=5 (Caused by ConnectTimeoutError(<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0xb45a4f30>, 'Connection to www.nzbclub.com timed out. (connect timeout=30)'))
[0m
05-05 03:12:43 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:12:43 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:12:43 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:12:43 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:12:43 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:12:43 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:12:45 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:12:45 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:13:13 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:13:13 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:13:13 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Star+Wars%3A+Episode+IX+2019%22/0/7/202, data: [][0m
05-05 03:13:14 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:13:15 INFO [0m[core.media.movie.searcher] Search for Star Wars: Episode IX in Screener ignoring ETA[0m
05-05 03:13:15 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.nzbclub.com/nzbrss.aspx?q=Star+Wars%3A+Episode+IX+2019&rpp=200&sp=1&ns=1&ig=1&st=5, data: [][0m
05-05 03:13:15 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt2527338&adv_nfo=on&xminsize=600&adv_age=1500&max=400&xmaxsize=1600&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:13:45 ERROR [31m[hpotato.core.plugins.base] Failed opening url in NZBClub: https://www.nzbclub.com/nzbrss.aspx?q=Star+Wars%3A+Episode+IX+2019&rpp=200&sp=1&ns=1&ig=1&st=5 Traceback (most recent call last):
ConnectTimeout: HTTPSConnectionPool(host='www.nzbclub.com', port=443): Max retries exceeded with url: /nzbrss.aspx?q=Star+Wars%3A+Episode+IX+2019&rpp=200&sp=1&ns=1&ig=1&st=5 (Caused by ConnectTimeoutError(<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0xb4a44cd0>, 'Connection to www.nzbclub.com timed out. (connect timeout=30)'))
[0m
05-05 03:13:45 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:13:45 INFO [0m[hpotato.core.plugins.base] Opening url: get https://hd4free.xyz/searchapi.php?apikey=&username=&imdbid=tt2527338&internal=False, data: [][0m
05-05 03:13:45 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:13:45 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:13:45 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:13:45 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2527338&search=Star+Wars%3A+Episode+IX+2019&user=&passkey=xxx[0m
05-05 03:13:46 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:13:46 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:13:46 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:14:10 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:14:12 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/s/star-wars-episode-ix-2019/se/desc/1/, data: [][0m
05-05 03:14:12 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/eta/tt7286456/, data: [][0m
05-05 03:14:13 INFO [0m[core.media.movie.searcher] Search for Joker in 720p ignoring ETA[0m
05-05 03:14:13 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.nzbclub.com/nzbrss.aspx?q=Joker+2019&rpp=200&sp=1&ns=1&ig=1&st=5, data: [][0m
05-05 03:14:13 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt7286456&adv_nfo=on&xminsize=3000&adv_age=1500&max=400&xmaxsize=10000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:14:13 INFO [0m[hpotato.core.plugins.base] Opening url: get https://api.nzbgeek.info/api?imdbid=7286456&apikey=xxx&t=movie&extended=1&cat=2000, data: [][0m
05-05 03:14:43 ERROR [31m[hpotato.core.plugins.base] Failed opening url in NZBClub: https://www.nzbclub.com/nzbrss.aspx?q=Joker+2019&rpp=200&sp=1&ns=1&ig=1&st=5 Traceback (most recent call last):
ConnectTimeout: HTTPSConnectionPool(host='www.nzbclub.com', port=443): Max retries exceeded with url: /nzbrss.aspx?q=Joker+2019&rpp=200&sp=1&ns=1&ig=1&st=5 (Caused by ConnectTimeoutError(<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0xb4a44930>, 'Connection to www.nzbclub.com timed out. (connect timeout=30)'))
[0m
05-05 03:14:43 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:14:43 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/gimmepeers/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:14:43 INFO [0m[hpotato.core.plugins.base] Opening url: get https://hd4free.xyz/searchapi.php?apikey=&username=&imdbid=tt7286456&internal=False, data: [][0m
05-05 03:14:43 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:14:43 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:14:43 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:14:47 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/1337x/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:14:50 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:14:50 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:14:51 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/anidex/results/torznab/api?apikey=xxx&t=search&cat=&q=?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:15:08 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:15:09 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/isohunt2/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:15:11 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:15:34 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:15:38 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/limetorrents/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:15:40 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/magnetdl/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:15:43 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/nyaasi/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:15:44 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/joker-2019/se/desc/1/, data: [][0m
05-05 03:15:44 INFO [0m[hpotato.core.plugins.base] Opening url: get https://yts.am/api/v2/list_movies.json?query_term=tt7286456&limit=10&page=1, data: [][0m
05-05 03:15:44 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Joker+2019%22/0/7/207, data: [][0m
05-05 03:15:44 INFO [0m[hpotato.core.plugins.base] Opening url: get http://katcr.to/span/m-i7286456/, data: [][0m
05-05 03:15:44 INFO [0m[hpotato.core.plugins.base] Opening url: get https://torrentapi.org/pubapi_v2.php?token=lt1jzcqb3x&mode=search&search_imdb=tt7286456&min_seeders=10&min_leechers=0&ranked=1&category=movies&format=json_extended&app_id=couchpotato, data: [][0m
05-05 03:15:44 ERROR [31m[hpotato.core.plugins.base] Failed opening url in Rarbg: https://torrentapi.org/pubapi_v2.php?token=lt1jzcqb3x&mode=search&search_imdb=tt7286456&min_seeders=10&min_leechers=0&ranked=1&category=movies&format=json_extended&app_id=couchpotato Traceback (most recent call last):
HTTPError: 429 Client Error:  for url: https://torrentapi.org/pubapi_v2.php?token=lt1jzcqb3x&mode=search&search_imdb=tt7286456&min_seeders=10&min_leechers=0&ranked=1&category=movies&format=json_extended&app_id=couchpotato
[0m
05-05 03:16:14 ERROR [31m[hpotato.core.plugins.base] Failed opening url in MagnetDL: http://www.magnetdl.com/j/joker-2019/se/desc/1/ Traceback (most recent call last):
ReadTimeout: HTTPConnectionPool(host='www.magnetdl.com', port=80): Read timed out. (read timeout=30)
[0m
05-05 03:16:14 INFO [0m[core.media.movie.searcher] Search for Joker in 1080p ignoring ETA[0m
05-05 03:16:14 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.nzbclub.com/nzbrss.aspx?q=Joker+2019&rpp=200&sp=1&ns=1&ig=1&st=5, data: [][0m
05-05 03:16:14 INFO [0m[hpotato.core.plugins.base] Opening url: get https://www.binsearch.info/index.php?q=tt7286456&adv_nfo=on&xminsize=4000&adv_age=1500&max=400&xmaxsize=20000&m=n&adv_sort=date&adv_col=on, data: [][0m
05-05 03:16:44 ERROR [31m[hpotato.core.plugins.base] Failed opening url in NZBClub: https://www.nzbclub.com/nzbrss.aspx?q=Joker+2019&rpp=200&sp=1&ns=1&ig=1&st=5 Traceback (most recent call last):
ConnectTimeout: HTTPSConnectionPool(host='www.nzbclub.com', port=443): Max retries exceeded with url: /nzbrss.aspx?q=Joker+2019&rpp=200&sp=1&ns=1&ig=1&st=5 (Caused by ConnectTimeoutError(<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0xb39b9830>, 'Connection to www.nzbclub.com timed out. (connect timeout=30)'))
[0m
05-05 03:16:44 INFO [0m[hpotato.core.plugins.base] Opening url: post https://www.sceneaccess.eu/login, data: ['username', 'password', 'submit'][0m
05-05 03:16:44 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:16:44 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:16:44 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt7286456&search=Joker+2019&user=&passkey=xxx[0m
05-05 03:16:44 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
05-05 03:16:45 ERROR [31m[edia._base.providers.base] Failed to login SceneAccess: unknown[0m
05-05 03:16:45 ERROR [31m[edia._base.providers.base] Failed to login to: SceneAccess[0m
05-05 03:16:45 INFO [0m[hpotato.core.plugins.base] Opening url: get http://www.magnetdl.com/j/joker-2019/se/desc/1/, data: [][0m
05-05 03:16:46 INFO [0m[chpotato.core._base._core] Shutting down[0m
05-05 03:16:46 INFO [0mShutting down thread pool[0m
